scripts/build-container-image.sh

```diff
@@ -47,22 +65,17 @@ git_get_current_commit_short_hash() {
   git rev-parse --short HEAD
 }
 
-git_get_current_branch() {
-  git rev-parse --abbrev-ref HEAD
-}
-
 get_date() {
   # Should this be current date, or latest commit date?
   date +%Y%m%d
 }
 
-get_image_tag() {
-  echo "$(git_get_current_branch)_$(get_date)_$(git_get_current_commit_short_hash)"
-}
-
 cleanup_old_tags() {
-  all_tags=$(list_tags)
-  branch=$(git_get_current_branch)
+  if [[ "${GIT_REFERENCE_TYPE}" != "branch" ]]; then
+    return
+  fi
+  all_tags=($(list_tags))
+  branch="${GIT_REFERENCE}"
 
   declare -a branch_tags=()
 
```

Comment:
L87 (GH doesn't allow me to comment on that, since it is not changed) is buggy as well.

```suggestion
  # shellcheck: disable=SC2207
  sorted_existing_tags=($(sort <<<"${branch_tags[*]}"))
```
Ie. remove set of `"`, which would make the sort output a single string, and remove the `IFS=$'\n'` since we're not line-feed delimited, but whitespace delimited.

Reply:
Sorry, the disable line should read `# shellcheck disable=SC2207` without the `:`